snuba/admin/audit_log/base.py

```diff
@@ -10,7 +10,9 @@
 
 class AuditLog:
     def __init__(self) -> None:
-        self.logger = structlog.get_logger()
+        self.logger = structlog.get_logger().bind(
```

Comment:
would be good to explain why the bind is important in a comment

Reply:
added